tests/vmi_configuration_test.go

```diff
@@ -1163,9 +1190,16 @@ var _ = Describe("[sig-compute]Configurations", func() {
 
 		Context("[rfe_id:893][crit:medium][vendor:cnv-qe@redhat.com][level:component]with rng", func() {
 			var rngVmi *v1.VirtualMachineInstance
+			var noRng = func() libvmi.Option {
```

Comment:
Can you make it a regular function?

Reply:
[Done](https://github.com/kubevirt/kubevirt/pull/7545/commits/2b87d96127b8269f527dcb322eb3ef9a11380210)